Mathlib/RingTheory/Flat/FaithfullyFlat.lean

```diff
@@ -25,6 +25,12 @@ A module `M` over a commutative ring `R` is *faithfully flat* if it is flat and
   is flat and tensoring with `M` is faithful, i.e. `N ≠ 0` implies `N ⊗ M ≠ 0`.
 - `Module.FaithfullyFlat.iff_flat_and_lTensor_faithful`: an `R`-module `M` is faithfully flat iff it
   is flat and tensoring with `M` is faithful, i.e. `N ≠ 0` implies `M ⊗ N ≠ 0`.
+- `Module.FaithfullyFlat.iff_iff_rTensor_exact`: an `R`-module `M` is faithfully flat iff tensoring
```

Comment:
I think this name is funny, but also kind of bad. Probably `.iff_exact_iff_rTensor_exact` is better (it just feels less confusing than `iff_iff`). Similarly `iff_iff_lTensor_exact`.